src/baldr/location.cc

```diff
@@ -4,10 +4,14 @@
 #include <boost/property_tree/json_parser.hpp>
 
 #include "baldr/location.h"
+#include "baldr/rapidjson_utils.h"
 #include "midgard/pointll.h"
 #include "midgard/logging.h"
 #include "midgard/util.h"
 
+#include <rapidjson/document.h>
```

Comment:
maybe common includes (like these?) could just be included in `baldr/rapidjson_utils.h` ?